internal/cli/atlas/clusters/watch.go

```diff
@@ -47,8 +46,8 @@ func (opts *WatchOpts) initStore(ctx context.Context) func() error {
 }
 
 func isRetryable(err error) bool {
-	var atlasErr *atlas.ErrorResponse
-	return errors.As(err, &atlasErr) && atlasErr.ErrorCode == "CLUSTER_NOT_FOUND"
+	atlasErr, ok := admin.AsError(err)
```

Comment:
can you create a ticket to do a grep for usages of `atlas.ErrorResponse` and fix them?

Reply:
I was fixing that as part of [CLOUDP-181568](https://jira.mongodb.org/browse/CLOUDP-181568)
https://github.com/mongodb/mongodb-atlas-cli/pull/1995

I will work on failing tests and merge the initial PR soon.